### ISSUE TITLE
pkgsMusl.linux-pam fails to build: patch musl-fix-pam_exec.patch out of date

### ISSUE THREAD

**Issue by nomeata** [NixOS/nixpkgs#108692]
With current master +  #108685 (fixing coreutils), the next issue I am facing is this:
```
~/build/nixpkgs $ nix-build -A pkgsMusl.linux-pam
these derivations will be built:
  /nix/store/0j569z807p9fjf0083xhzhjpps4vwc9h-linux-pam-1.5.1.drv
building '/nix/store/0j569z807p9fjf0083xhzhjpps4vwc9h-linux-pam-1.5.1.drv'...
unpacking sources
unpacking source archive /nix/store/cbbam7s59sqrsqk2dh25izkjmdngwd5h-Linux-PAM-1.5.1.tar.xz
source root is Linux-PAM-1.5.1
setting SOURCE_DATE_EPOCH to timestamp 1606323454 of file Linux-PAM-1.5.1/doc/specs/parse_y.h
patching sources
applying patch /nix/store/mbcmh0dbsczh53p7h5mcwbmynx1fqq1m-fix-compat.patch?id=05a62bda8ec255d7049a2bd4cf0fdc4b32bdb2cc
patching file modules/pam_lastlog/pam_lastlog.c
Hunk #2 succeeded at 52 with fuzz 1 (offset 3 lines).
applying patch /nix/store/nynvcd4ljflkxw6gb81dqdwwvidixzvg-libpam-fix-build-with-eglibc-2.16.patch?id=05a62bda8ec255d7049a2bd4cf0fdc4b32bdb2cc
patching file modules/pam_unix/pam_unix_acct.c
Hunk #1 succeeded at 50 (offset 3 lines).
applying patch /nix/store/r1f2scxc864wdy3r9mm1lx7gsy36ml0c-musl-fix-pam_exec.patch
patching file modules/pam_exec/pam_exec.c
Hunk #1 FAILED at 103.
Hunk #2 FAILED at 180.
2 out of 2 hunks FAILED -- saving rejects to file modules/pam_exec/pam_exec.c.rej
builder for '/nix/store/0j569z807p9fjf0083xhzhjpps4vwc9h-linux-pam-1.5.1.drv' failed with exit code 1
error: build of '/nix/store/0j569z807p9fjf0083xhzhjpps4vwc9h-linux-pam-1.5.1.drv' failed
```

This patch was added in “c260a8d0635 libpam: fix w/musl after update to 1.3.1” by @dtzWill.

Possibly broken by “d703c1ef25775dbe94e9120fe988db24c2df48c3 pam: 1.3.1 -> 1.5.1” by @NeQuissimus.

**Comment by nomeata** [comment]
Ok, looking at the file, it seems simply to be applied upstream alreay in https://github.com/linux-pam/linux-pam/commit/91d4678388b2a7d768ee2ec8cc569e11fc223ffd

**Comment by pjjw** [comment]
bleh i just leeroy jenkinsed in with my own patch, not seeing this, forgetting how often patches just sit in the PR queue, but this got fixed and merged in #109906

**Comment by nomeata** [comment]
Nothing to be sorry about, as long as it gets fixed :-)
